ddterm/shell/panelicon.js

```diff
@@ -47,7 +47,7 @@ const PanelIconBase = GObject.registerClass({
 
         this.add_child(new St.Icon({
             icon_name: 'utilities-terminal',
-            style_class: 'system-status-icon',
+            style_class: 'system-status-icon indicator-icon',
```

Comment:
Isn't there a built-in (Shell) style that does this? When other extensions add icons to the panel, is spacing similar to ddterm?

Reply:
here:

https://github.com/oae/gnome-shell-pano/blob/d0d71f8015ada661f35a1390502c8e09fa2a10b6/src/components/indicator/settingsMenu.ts#L62

Reply:
If we don't modify ddterm, using [Just Perfection](https://extensions.gnome.org/extension/3843/just-perfection/) will not achieve the desired effect and will result in inconsistent spacing between icons

Reply:
> If we don't modify ddterm, using [Just Perfection](https://extensions.gnome.org/extension/3843/just-perfection/) will not achieve the desired effect

What is "the desired effect"?

> will result in inconsistent spacing between icons

Current implementation is consistent with "Removable Drive Menu" from gnome-shell-extensions package: https://github.com/GNOME/gnome-shell-extensions/blob/4a841dfd4977c3d497b0241c7d0016c4290b3e07/extensions/drive-menu/extension.js#L144-L147

Reply:
I test again, you are right